descriptors/ModuleDescriptor-template.json

```diff
@@ -243,7 +249,11 @@
           "pathPattern": "/orders/pieces",
           "permissionsRequired": ["orders.pieces.item.post"],
           "modulePermissions": [
-            "orders-storage.pieces.item.post"
+            "orders-storage.pieces.item.post",
+            "orders-storage.po-lines.collection.get",
+            "acquisitions-units-storage.units.collection.get",
+            "acquisitions-units-storage.memberships.collection.get",
+            "orders-storage.acquisitions-unit-assignments.collection.get"
```

Comment:
`orders-storage.acquisitions-unit-assignments.*` are going to be removed in https://github.com/folio-org/mod-orders-storage/pull/146

Reply:
removed